src/Chapter1.hs

```diff
@@ -632,7 +643,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n
+  | n >= 0 && n < 10 = n
+  | otherwise =
+    let one = mod (abs n) 10
+        two = div (mod (abs n) 100) 10
```

Comment:
This case actually covers the previous one, but yours would be faster on small numbers 👍🏼

Reply:
thank you, I've updated it to
```
sumLast2 :: Int -> Int
sumLast2 n = 
  let a = abs n
      b = mod (div a 10) 10
      c = mod a 10
      d = b + c
  in d
```